frontend/providers/dbprovider/src/components/ErrorLog/LogTable.tsx

```diff
@@ -0,0 +1,351 @@
+              width={'100%'}
+              pl={'0px'}
+              p={'0'}
+              alignItems={'flex-start'}
+              gap={'0px'}
+              alignSelf={'stretch'}
+              borderBottom={'none'}
+            >
+              {podDisplay}
+            </Box>
+          );
+        }
+      }
+    ],
+    [t, pvcMap]
+  );
+
+  const table = useReactTable({
+    data,
+    columns,
+    getCoreRowModel: getCoreRowModel(),
+    getFilteredRowModel: getFilteredRowModel(),
+    state: {
+      globalFilter
+    },
+    onGlobalFilterChange,
+    globalFilterFn: (row, columnId, filterValue) => {
+      const timestamp = formatTime(row.original.timestamp, 'YYYY-MM-DD HH:mm:ss.SSS')
+        .toLowerCase()
+        .includes(filterValue.toLowerCase());
+      const content = row.original.content.toLowerCase().includes(filterValue.toLowerCase());
+      return timestamp || content;
+    }
+  });
+
+  return (
+    <Flex flexDir={'column'} w={'100%'} h={'100%'}>
+      <Flex alignItems={'center'} gap={4} justifyContent={'space-between'}>
+        <Flex alignItems={'center'} gap={4}>
+          <Text
+            sx={{
+              bg: 'transparent',
+              border: 'none',
+              boxShadow: 'none',
+              color: '#000',
+              fontFamily: 'Geist',
+              fontWeight: 500,
+              fontSize: '18px',
+              lineHeight: '28px',
+              fontStyle: 'normal'
+            }}
+          >
+            Logs
+          </Text>
+        </Flex>
+        {onExportLogs && (
+          <Button
+            onClick={onExportLogs}
+            leftIcon={<Upload size={16} style={{ flexShrink: 0 }} strokeWidth="2" />}
+            sx={{
+              display: 'flex',
+              height: '40px',
+              padding: '8px 16px',
+              justifyContent: 'center',
+              alignItems: 'center',
+              gap: '8px',
+              borderRadius: '8px',
+              border: '1px solid #E4E4E7',
+              background: '#FFF',
+              boxShadow: '0 1px 2px 0 rgba(0, 0, 0, 0.05)',
+              color: '#18181B',
+              fontFamily: 'Geist',
+              fontSize: '14px',
+              fontWeight: 500,
+              lineHeight: '20px',
+              fontStyle: 'normal',
+              '& svg': {
+                color: '#737373'
+              },
+              '&:hover': {
+                backgroundColor: '#000 !important',
+                color: '#FFF !important',
+                '& svg': {
+                  color: '#FFF !important'
+                }
+              }
+            }}
+          >
+            Export
+          </Button>
+        )}
+      </Flex>
+
+      {/* 分割线 */}
+      <Box
+        width={'100%'}
+        height={'1px'}
+        borderTop={'1px solid'}
+        borderTopColor={'grayModern.200'}
+        mt={'7px'}
+        mb={'20px'}
+      />
+
+      {data.length > 0 ? (
+        <>
+          <BaseTable
```

Comment:
This is a lot of !important overrides. It's generally best to avoid these if we can, just because they make the CSS super rigid.